### PR TITLE
Explain += in Creating Functions - Issue#529

### PR DESCRIPTION
**Pull request by mzc9**

### CODE REVIEW

**Comment by annefou** [comment]
Thanks @mzc9 for your contribution. We should probably refer to in-place operators as done in lesson 05 (http://swcarpentry.github.io/python-novice-inflammation/05-cond/index.html) with a link to http://swcarpentry.github.io/python-novice-inflammation/reference/#in-place-operators
Thanks.

**Comment by maxim-belkin** [comment]
`+=` is first used in episode 04 (`composite_data += data`). So, let's make sure we introduce `+=` in the right place.

Also, please estimate the amount of time it would require an instructor to go through the introduced material.

File: _episodes/06-func.md [hunk 1]
```diff
@@ -685,6 +685,25 @@ programmer. If you need to revisit code that you wrote months ago and
 haven't thought about since then, you will appreciate the value of
 readable code!
 
+Python (and most other languages in the C family) provides
```
Comment (maxim-belkin) [inline _episodes/06-func.md]: Remove `(and most other languages in the C family)`

File: _episodes/06-func.md [hunk 2]
```diff
@@ -685,6 +685,25 @@ programmer. If you need to revisit code that you wrote months ago and
...
+> ~~~
+> {: .language-python}
+>
+> ~~~
+> 6
+> ~~~
+> {: .output}
+
+
+
```
Comment (maxim-belkin) [inline _episodes/06-func.md]: Remove 1 empty line

File: _episodes/06-func.md [hunk 3]
```diff
@@ -685,6 +685,25 @@ programmer. If you need to revisit code that you wrote months ago and
 haven't thought about since then, you will appreciate the value of
 readable code!
 
+Python (and most other languages in the C family) provides
+[in-place operators]({{ page.root }}/reference/#in-place-operators)
+that work like this:
```
Comment (maxim-belkin) [inline _episodes/06-func.md]: replace `work like this:` with an explanation.
Note, the one given on the reference page is not ideal either. Explain what `+=` does in simple terms.

File: _episodes/06-func.md [hunk 4]
```diff
@@ -685,6 +685,25 @@ programmer. If you need to revisit code that you wrote months ago and
...
+>
+> ~~~
+> x = 1  # original value
+> x += 1 # add one to x, assigning result back to x
```
Comment (maxim-belkin) [inline _episodes/06-func.md]: > x += 1 # increment x by 1

**Comment by maxim-belkin** [comment]
ping, @mzc9

**Comment by ldko** [comment]
This explanation is given in a challenge in the [conditionals episode](https://swcarpentry.github.io/python-novice-inflammation/07-cond/index.html). Closing this due to that and the PR now being stale.

**Pull request closed without merging**